plugins/source/aws/resources/services/emr/studios.go

```diff
@@ -0,0 +1,65 @@
+package emr
+
+import (
+	"context"
+	"github.com/apache/arrow/go/v13/arrow"
+	"github.com/aws/aws-sdk-go-v2/service/emr"
+	"github.com/aws/aws-sdk-go-v2/service/emr/types"
+	"github.com/cloudquery/cloudquery/plugins/source/aws/client"
+	"github.com/cloudquery/plugin-sdk/v4/schema"
+	"github.com/cloudquery/plugin-sdk/v4/transformers"
+)
+
+func Studios() *schema.Table {
+	tableName := "aws_emr_studios"
+	return &schema.Table{
+		Name:                tableName,
+		Description:         `https://docs.aws.amazon.com/emr/latest/APIReference/API_Studio.html`,
+		Resolver:            fetchEmrStudios,
+		PreResourceResolver: getStudio,
+		Multiplex:           client.ServiceAccountRegionMultiplexer(tableName, "elasticmapreduce"),
+		Transform:           transformers.TransformWithStruct(&types.Studio{}),
+		Columns: []schema.Column{
+			client.DefaultAccountIDColumn(false),
+			client.DefaultRegionColumn(false),
+			{
+				Name:       "studio_id",
+				Type:       arrow.BinaryTypes.String,
+				Resolver:   schema.PathResolver("StudioId"),
+				PrimaryKey: true,
+			}},
```

Comment:
In `aws_emr_studio_session_mapping` we use both `account_id` and `region` as PKs, but here we only use `studio_id`... If `studio_id` is globally unique (across all regions and accounts) we should remove those 2 PKs from the relation table, otherwise maybe we should add Region+Account to this PK?

Reply:
Good point. I _think_ `studio_id` would be unique, but the [documentation](https://docs.aws.amazon.com/cli/latest/reference/emr/create-studio.html#output) doesn't make it completely clear. In this scenario do you think it would be better to add the 2PKs to this table also?

Reply:
We only ever assume an ID is going to be unique within a single region + account unless AWS clearly documents something different... For example s3 bucket name is guaranteed to be globally unique... So it is better to add the region + account ID as part of the PK